MicrosoftActiveDirectory/tests/test_search.py

```diff
@@ -112,3 +115,36 @@ def test_ldap_action_serialization():
         orjson.dumps(results)
     except (TypeError, ValueError) as e:
         assert False, f"Serialization failed: {str(e)}"
+
+def test_search_to_file():
```

Comment:
In the `conftest.py` file, add the following code (see [DUO 's conftest.py](https://github.com/SEKOIA-IO/automation-library/blob/main/Duo/tests/conftest.py)):
```python
from shutil import rmtree
from tempfile import mkdtemp

import pytest
from sekoia_automation import constants


@pytest.fixture
def data_storage():
    original_storage = constants.DATA_STORAGE
    constants.DATA_STORAGE = mkdtemp()

    yield constants.DATA_STORAGE

    rmtree(constants.DATA_STORAGE)
    constants.DATA_STORAGE = original_storage
```

Update `configured_action` (L10) like the folllowing code:
```python
def configured_action(action: MicrosoftADAction, data_path: Path | None = None):
    module = MicrosoftADModule()
    a = action(module, data_path=data_path)
    
    a.module.configuration = {
        "servername": "test_servername",
        "admin_username": "test_admin_username",
        "admin_password": "test_admin_password",
    }
    
    return a
```

Don't forget to import `Path`, at the begining of the file:
```python
from pathlib import Path
```

```suggestion
def test_search_to_file(data_storage):
```